security.rst

```diff
@@ -14,9 +14,9 @@ is both flexible and (hopefully) fun to work with.
 Since there's a lot to talk about, this article is organized into a few big
 sections:
 
-#. Initial ``security.yml`` setup (*authentication*);
+#. Initial ``security.yml`` setup (*authentication*).
```

Comment:
At first I also thought this was a mistake ... but apparently is common in English to use a comma after each list element, except the last one. We use that style in the rest of the docs, so we shouldn't change this. Thanks!